make/autoconf/flags-ldflags.m4

```diff
@@ -21,10 +21,13 @@
 # Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
 # or visit www.oracle.com if you need additional information or have any
 # questions.
-#
+# 
```

Comment:
Please remove the trailing space and add the requested blank line after line 27.